content/en/docs/instrumentation/erlang/resources.md

```diff
@@ -88,7 +88,7 @@ flattened and combined with `.`, so
 ## Custom resource detectors
 
 Custom resource detectors can be created by implementing the
-[`otel_resource_detector` behaviour](https://hexdocs.pm/opentelemetry/1.3.0/otel_resource_detector.html#callbacks)
+[`otel_resource_detector` behavior](https://hexdocs.pm/opentelemetry/1.3.0/otel_resource_detector.html#callbacks)
```

Comment:
while both spellings are acceptable, the Erlang standard documentation and language favor the `behaviour` spelling pretty much everywhere, as in the following pages:

- https://www.erlang.org/doc/reference_manual/modules.html#behaviour-module-attribute
- https://www.erlang.org/doc/design_principles/des_princ.html#behaviours
- https://www.erlang.org/doc/design_principles/sup_princ.html

Reply:
[This document](https://www.erlang.org/doc/reference_manual/modules.html#behaviour-module-attribute) says

> The spelling `behavior` is also accepted.

At the end it's British vs American English and so far we use American English consistently in the documentation. 

I'm with @chalin here and would vote for "behavior" to be consistent, but since this is erlang specific and I lack the subject matter expertise here what the audience of the erlang community expects, I'll defer this to @open-telemetry/erlang-approvers 

We can also highlight that this is a language specific term and include it in the backticks:
```
[`otel_resource_detector behaviour`](https://hexdocs.pm/opentelemetry/1.3.0/otel_resource_detector.html#callbacks)
```

rendering to

[`otel_resource_detector behaviour`](https://hexdocs.pm/opentelemetry/1.3.0/otel_resource_detector.html#callbacks)